pilot/const/function_calls.py

```diff
@@ -200,29 +200,48 @@ def command_definition(description_command='A single command that needs to be ex
                             'properties': {
                                 'type': {
                                     'type': 'string',
-                                    'enum': ['command', 'code_change', 'human_intervention'],
+                                    'enum': ['command', 'save_file', 'modify_file', 'human_intervention'],
```

Comment:
I renamed `code_change` to `save_file` to avoid LLM being confused what `code_change` should be used for. The `modify_file` one is new.